configs/config.yaml

```diff
@@ -0,0 +1,34 @@
+train:
+  batch_size: 64
+  epochs: 5
+  log_interval: 100
+  generator_lr: 1e-4
+  discriminator_lr: 1e-5
+  num_workers: 4
+  # for continuing training
+  load_checkpoint: # change from None to checkpoint path to continue training
+  more_epochs: 20 # How many more epochs to train
+
+logger:
+  checkpoint_path: "checkpoints/"
+  chart_path: "tmp/"
+
+system:
+  device: "cuda:0"
+  seed: 23
+
+data:
+  channels: 1
+  size: 250
```

Comment:
I tried running with `size=512` but got this:

```
Error executing job with overrides: ['data.size=512']
Traceback (most recent call last):
  File "/home/hagrid/workson/ecg-gan/train.py", line 208, in main
    train_step(
  File "/home/hagrid/workson/ecg-gan/train.py", line 91, in train_step
    discriminator_error_real = criterion(disc_real_output, label)
  File "/home/hagrid/.virtualenvs/ecg-gan/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1501, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/hagrid/.virtualenvs/ecg-gan/lib/python3.10/site-packages/torch/nn/modules/loss.py", line 619, in forward
    return F.binary_cross_entropy(input, target, weight=self.weight, reduction=self.reduction)
  File "/home/hagrid/.virtualenvs/ecg-gan/lib/python3.10/site-packages/torch/nn/functional.py", line 3089, in binary_cross_entropy
    raise ValueError(
ValueError: Using a target size (torch.Size([64])) that is different to the input size (torch.Size([1984])) is deprecated. Please ensure they have the same size.
```

Reply:
Ah sorry for that, currently its a little hard-coded towards 125, 250, 500, 1000 so that its possible to divide the dataset, the generator then cuts what is above this value. In case of 500, it generates size 512 and we cut the remaining 12. I could have been more precise regarding the powers of 2